SwiftVideoBackground/Sources/VideoBackground.swift

```diff
@@ -96,131 +99,126 @@ public class VideoBackground {
     ///     - isMuted: Bool indicating whether video is muted. Defaults to true.
     ///     - alpha: CGFloat between 0 and 1. The higher the value, the darker the video. Defaults to 0.
     ///     - willLoopVideo: Bool indicating whether video should restart when finished. Defaults to true.
+    ///     - setAudioSessionAmbient: Bool indicating whether to set the shared `AVAudioSession` to ambient. If this is
+    ///         not done, audio played from your app will pause other audio playing on the device. Defaults to true.
     /// - Throws: `VideoBackgroundError.videoNotFound` if the video cannot be found.
     public func play(view: UIView,
                      name: String,
                      type: String,
                      isMuted: Bool = true,
                      alpha: CGFloat = 0,
-                     willLoopVideo: Bool = true) throws {
-        try play(
+                     willLoopVideo: Bool = true,
+                     setAudioSessionAmbient: Bool = true) throws {
+        guard let path = Bundle.main.path(forResource: name, ofType: type) else {
+            throw VideoBackgroundError.videoNotFound(VideoInfo(name: name, type: type))
+        }
+        let url = URL(fileURLWithPath: path)
+        play(
             view: view,
-            videoInfos: [VideoInfo(name: name, type: type)],
+            url: url,
             isMuted: isMuted,
             alpha: alpha,
-            willLoopVideo: willLoopVideo
+            willLoopVideo: willLoopVideo,
+            setAudioSessionAmbient: setAudioSessionAmbient
         )
     }
 
-    /// Plays videos one after another on a UIView.
+    /// Plays a video.
     ///
     /// - Parameters:
     ///     - view: UIView that the video will be played on.
-    ///     - videoInfos: Array of `VideoInfo` for the videos to be played.
+    ///     - url: URL of the video. Can be from your local file system or the web.
     ///     - isMuted: Bool indicating whether video is muted. Defaults to true.
     ///     - alpha: CGFloat between 0 and 1. The higher the value, the darker the video. Defaults to 0.
     ///     - willLoopVideo: Bool indicating whether video should restart when finished. Defaults to true.
-    /// - Throws: `VideoBackgroundError.videoNotFound` if a video cannot be found.
+    ///     - setAudioSessionAmbient: Bool indicating whether to set the shared `AVAudioSession` to ambient. If this is
+    ///         not done, audio played from your app will pause other audio playing on the device. Defaults to true.
     public func play(view: UIView,
-                     videoInfos: [VideoInfo],
+                     url: URL,
                      isMuted: Bool = true,
                      alpha: CGFloat = 0,
-                     willLoopVideo: Bool = true) throws {
+                     willLoopVideo: Bool = true,
+                     setAudioSessionAmbient: Bool = true) {
```

Comment:
What happens if the URL (external web) doesn't work? Should we make this throw as well?

Reply:
Nothing happens if the url points to a non video. Good point, I will investigate on how to make this throw

Reply:
I don't see any kind of error handling regarding URLs with the native structures I'm using. I will add notes in docs.

Reply:
For reference I am using `AVPlayer(url:)`.

Reply:
✅